index.js

```diff
@@ -149,8 +158,6 @@ module.exports = [...compat.extends('airbnb-base'),
         },
 
         rules: {
-            // TypeScript doesn't support imports with `.ts` extensions, and `.js` makes the rule complain.
-            'import/extensions': 'off',
```

Comment:
- TS *does* support importing with `.ts` but you'll need to enable two separate compiler options (one to allow it and one to rewrite it to .js)
- disabling this in TS completely means we'll end up having the same import issues in any TS ESM repo... Is there no alternative for it in ts? Maybe configuring it like this https://github.com/iCrawl/eslint-config-neon/blob/69545e180de275176b55039f615a754d0d2de53d/src/typescript.ts#L378 (and switching from the import plugin to import-x?)

Reply:
Not sure I follow. I enabled it for TS as well

Reply:
I *love* the totally obvious difference between red and green on the mobile app (not). My bad then 😅

Reply:
No worries 😉